services/src/main/resources/config-spring-geonetwork.xml

```diff
@@ -164,4 +164,36 @@
 
   <bean id="userFeedbackService" class="org.fao.geonet.api.userfeedback.service.UserFeedbackDatabaseService"/>
 
+  <util:map id="translationPacks">
```

Comment:
Can be this added in the `web` project or at least would be possible to override it? Thinking about custom schemas.

Reply:
I suppose we can override the bean config from somewhere else. No tested.